db/migrate/20151231125222_extract_access_limiting.rb

```diff
@@ -0,0 +1,21 @@
+class ExtractAccessLimiting < ActiveRecord::Migration
+  def up
+    create_table :access_limits do |t|
+      t.integer :target_id, null: false
+      t.string :target_type, null: false
+      t.json :users, null: false, default: []
+      t.timestamps null: false
+    end
+
+    DraftContentItem.where("access_limited::text <> '{}'::text").each do |limited_draft|
+      AccessLimit.create(
```

Comment:
#create! might be a good idea here incase we add validations later.

Printing as well?